migration-tool/cmd/prom-migrator/main.go

```diff
@@ -35,7 +35,7 @@ const (
 	defaultStartTime       = "1970-01-01T00:00:00+00:00" // RFC3339 based time.Unix from 0 seconds.
 	defaultMaxReadDuration = time.Hour * 2
 	defaultLaIncrement     = time.Minute
-	version                = "0.0.5"
+	version                = "0.0.6"
```

Comment:
Why are we bumping prom-migrator (i believe it should have not dependency on Promscale)?

Reply:
@alejandrodnm did few changes in prom-migrator[1], hence I bumped it's version

[1] https://github.com/timescale/promscale/commits/master/migration-tool